README.md

```diff
@@ -137,34 +139,54 @@ The dependency keys can be any strings, but it's recommended to use package name
 
 Examples:
 
-```js
+```json
 // Clone 'git@github.com:cksource/foo.git' and check out to 'master'.
 {
     "foo": "git@github.com:cksource/foo.git"
 }
 ```
 
-```js
+```json
 // Short format. Clone 'git@github.com:cksource/foo.git' and check out to branch 'dev'.
 {
     "@cksource/foo": "cksource/foo#dev"
 }
 ```
 
-```js
+```json
 // Clone 'https://github.com/cksource/foo.git' (via HTTPS) and check out to tag 'v1.2.3'.
 {
     "foo": "https://github.com/cksource/foo.git@v1.2.3"
 }
 ```
 
-```js
+```json
 // Clone 'cksource/foo' and check out to the latest available tag.
 {
     "foo": "cksource/foo@latest"
 }
 ```
 
+### The `presets` option
+
+This option allows the user to create an easy switch between different states of the dependencies. When using any command with the `--preset` option, it will behave as if the `dependencies` option was using values from the given preset. Dependencies that are not defined in the preset but are defined in the `dependencies`, will use version from the `dependencies` as a fallback.
+
+Example:
+
+```json
+{
+    "presets": {
+        "dev": {
+            "@cksource/foo": "cksource/foo#dev"
+        },
+        "newFeature": {
+            "@cksource/foo": "cksource/foo#newFeatureBranch",
+            "@cksource/bar": "cksource/foo#newFeatureBranch"
+        }
```

Comment:
Not sure if it's a standard, but CLI apps use kebab-case. Hence, I'd suggest renaming the preset name to something significant, e.g., `stable` or `example-feature`. Also, we do not follow camelCase syntax when defining a branch name. I'd go with, e.g., `i/1-example-feature`.